aip/0123.md

```diff
@@ -72,12 +73,13 @@ message Topic {
 - Patterns **must** correspond to the [resource name][aip-122].
 - Pattern variables (the segments within braces) **must** use `snake_case`, and
   **must not** use an `_id` suffix.
-- Pattern variables (the segments within braces) **must** begin with a
-lower-cased letter and contain only ASCII letters, numbers, and underscore
-(`/[a-z][a-zA-Z0-9_]*/`).
 
 #### Complex resource ID path segments
 
+Note: Complex resource ID path segments are supported for historical reasons,
+and should not be used in new APIs. See [AIP-124][] for advice on handling
```

Comment:
The "should not" should probably be in bold.

Reply:
Good point, done.